packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -12,7 +12,12 @@ import 'package:flutter/material.dart';
 void main() {
   group('$ReorderableListView', () {
     const double itemHeight = 48.0;
-    const List<String> originalListItems = <String>['Item 1', 'Item 2', 'Item 3', 'Item 4'];
+    const List<String> originalListItems = <String>[
```

Comment:
I think you might have accidentally run `dartfmt` on the `_test.dart` file. Could you revert these formatting changes?